src/View/OutputView.js

```diff
@@ -0,0 +1,29 @@
+const { Console } = require('@woowacourse/mission-utils');
+const { OUTPUT, MESSAGE, STRIKE_COUNT } = require('../libs/Const');
+
+class OutputView {
+  static printMessage(message) {
+    Console.print(message);
+  }
+
+  static printResult({ ballCount, strikeCount }) {
+    if (ballCount === 0 && strikeCount === 0) {
+      Console.print(OUTPUT.nothing);
+      return;
+    }
+
+    this.#printBallAndStrikeCount(ballCount, strikeCount);
+
+    if (strikeCount === STRIKE_COUNT) Console.print(MESSAGE.success);
+  }
+
+  static #printBallAndStrikeCount(ballCount, strikeCount) {
+    const ballCountPrint = ballCount !== 0 ? `${ballCount}${OUTPUT.ball} ` : '';
+    const strikeCountPrint =
+      strikeCount !== 0 ? `${strikeCount}${OUTPUT.strike}` : '';
+
+    Console.print(`${ballCountPrint}${strikeCountPrint}`);
```

Comment:
이렇게 출력하면 볼만 있을 때, '2볼'이 아닌 '2볼 '로 출력이 되니까, trim()으로 한 번 더 최종 출력을 정제하면 좋을 것 같습니다 !!
```suggestion
    Console.print(`${ballCountPrint}${strikeCountPrint}`.trim());
```